sdks/java/extensions/euphoria/euphoria-beam/src/main/java/org/apache/beam/sdk/extensions/euphoria/beam/BeamFlow.java

```diff
@@ -89,6 +103,30 @@ public static BeamFlow create(String name, Pipeline pipeline) {
     return new BeamFlow(name, pipeline);
   }
 
+  /**
+   * Create flow from input {@link PCollection} which is
+   * {@linkplain PCollection#isFinishedSpecifying() finished specifying}.
+   *
+   *
+   * @param pCollection the input {@link PCollection} to wrap into new flow
+   * @return constructed flow
+   */
+  public static <T> BeamFlow create(PCollection<T> pCollection) {
```

Comment:
should this be named `of` instead to be consistent with the rest of the api?

Reply:
I agree. Requested change added to [BEAM-4428](https://issues.apache.org/jira/browse/BEAM-4428).